pycaret/containers/models/classification.py

```diff
@@ -592,7 +596,18 @@ def __init__(self, experiment):
         }
         tune_args = {}
         tune_grid = {
-            "max_iter_predict": [100, 200, 300, 400, 500, 600, 700, 800, 900, 1000,]
+            "max_iter_predict": [
```

Comment:
Would recommend just removing the `,` after the 1000. That should cause `black` to reformat it back to a single line.

Reply:
Seems to have fixed it!